smithy-base/src/main/java/software/amazon/smithy/gradle/tasks/SmithyBuildTask.java

```diff
@@ -104,10 +106,27 @@ List<String> getModelAbsolutePaths() {
                 .collect(Collectors.toList());
     }
 
+    /**
+     * Read-only property.
+     *
+     * @return Returns true if at least one of the specified build configs exists.
+     */
+    @Internal
+    Provider<Boolean> getSmithyBuildConfigsMissing() {
+        return getSmithyBuildConfigs().map(
+                files -> !files.isEmpty() && files.filter(File::exists).isEmpty()
+        );
+    }
+
     @TaskAction
     public void execute() {
         writeHeading("Running smithy build");
 
+        if (getSmithyBuildConfigsMissing().get()) {
+            throw new GradleException("No smithy-build configs found. "
+                    + "If this was intentional, set the `smithyBuildConfigs` property to an empty list.");
+        }
```

Comment:
The comment for getSmithyBuildConfigs says it returns true if one of the specified build configs exists, but when it is used, the condition isn't negated, so I think it would throw?

Reply:
Or I don't understand the logic of that method

Reply:
The comment is incorrect. I will update it.